cmds/exp/ipmidump/ipmidump.go

```diff
@@ -194,3 +207,112 @@ func selInfo() {
 		}
 	}
 }
+
+func lanConfig() {
+	const (
+		setInProgress byte = iota
+		_
+		_
+		IPAddress
+		IPAddressSrc
+		MACAddress
+	)
+
+	setInProgressStr := []string{
+		"Set Complete", "Set In Progress", "Commit Write", "Reserved",
+	}
+
+	IPAddressSrcStr := []string{
+		"Unspecified", "Static Address", "DHCP Address", "BIOS Assigned Address",
+	}
+
+	ipmi, err := ipmi.Open(0)
+	if err != nil {
+		fmt.Printf("Failed to open ipmi device: %v\n", err)
```

Comment:
This is going to get a null pointer error as soon as you use ipmi. Since this function does not return an error you might as well call log.Fatal(err) instead. The Error should contain all the needed information.

Reply:
Modified to call log.Fatal(err) instead.